pydantic/fields.py

```diff
@@ -239,11 +252,10 @@ def Field(
     decimal_places: int = None,
     min_items: int = None,
     max_items: int = None,
-    unique_items: bool = None,
     min_length: int = None,
     max_length: int = None,
-    allow_mutation: bool = True,
-    regex: str = None,
+    frozen: bool = None,
+    pattern: str = None,
     discriminator: str = None,
     repr: bool = True,
     **extra: Any,
```

Comment:
This makes a lot of sense. It would also be easy to put through a deprecation cycle if necessary (just merging the dicts and emitting a warning if `**extra` is not empty)

Reply:
Totally agree, done.

Reply:
Amazing, thanks!